x-pack/test_serverless/functional/test_suites/search/search_index_detail.ts

```diff
@@ -54,7 +54,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         await pageObjects.svlSearchIndexDetailPage.expectConnectionDetails();
       });
 
-      it.skip('should show api key', async () => {
+      it('should show api key', async () => {
```

Comment:
could you break out the api key tests into its own describe so that any flakiness that may happen with api-keys, they are only skipped?

Reply:
Yes, I can